post_installation_configuration/configuring-multi-arch-compute-machines/creating-multi-arch-compute-nodes-ibm-power.adoc

```diff
@@ -0,0 +1,21 @@
+:_content-type: ASSEMBLY
+:context: creating-multi-arch-compute-nodes-ibm-power
+[id="creating-multi-arch-compute-nodes-ibm-power"]
+= Creating a cluster with multi-architecture compute machine on IBM Power
+include::_attributes/common-attributes.adoc[]
+
+toc::[]
+
+To create a cluster with multi-architecture compute machines on {ibmpowerProductName}(`ppc64le`) with {op-system-base} KVM, you must have an existing single-architecture (`x86_64`) cluster. For more information on {ibmpowerProductName} installations, see xref:../../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metal[Installing a user provisioned cluster on IBM Power]. You can then add `ppc64le` compute machines to your {product-title} cluster.
```

Comment:
```suggestion
To create a cluster with multi-architecture compute machines on {ibmpowerProductName}(`ppc64le`), you must have an existing single-architecture bare metal (`x86_64`) cluster. For more information on bare metal installations, see xref:../../installing/installing_bare_metal/installing-bare-metal.adoc#installing-bare-metalInstalling a user provisioned cluster on bare meta]. You can then add `ppc64le` compute machines to your {product-title} cluster.
```

Reply:
Reverting my suggestion, after discussion with SMEs for IBM Z we hae removed the "bare metal" wording in my [PR](https://github.com/openshift/openshift-docs/pull/63855)